### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by flands**
- Add a fancy header
- Add contributing information

### CODE REVIEW

**Review by iNikem — APPROVED** [review]
Thanks!

File: README.md [hunk 1]
```diff
@@ -1,12 +1,44 @@
...
+    <a href="https://github.com/open-telemetry/community#special-interest-groups">Getting Involved<a/>
+    &nbsp;&nbsp;&bull;&nbsp;&nbsp;
+    <a href="https://gitter.im/open-telemetry/opentelemetry-java-instrumentation">Getting In Touch<a/>
+  </strong>
+</p>
+
+<p align="center">
+  <a href="https://goreportcard.com/report/github.com/open-telemetry/opentelemetry-java-instrumentation">
+    <img alt="Go Report Card" src="https://goreportcard.com/badge/github.com/open-telemetry/opentelemetry-java-instrumentation?style=for-the-badge">
+  </a>
```
Comment (mateuszrzeszutek) [inline README.md:16]: > There was an error processing your request: Could not analyze the repository: no .go files found

I don't think that we need this one - there are no go files in this repo 😄

File: README.md [hunk 2]
```diff
@@ -1,12 +1,44 @@
...
+  <a href="https://circleci.com/gh/open-telemetry/opentelemetry-java-instrumentation">
+    <img alt="Build Status" src="https://img.shields.io/circleci/build/github/open-telemetry/opentelemetry-java-instrumentation?style=for-the-badge">
+  </a>
+  <a href="https://codecov.io/gh/open-telemetry/opentelemetry-java-instrumentation/branch/master/">
+    <img alt="Codecov Status" src="https://img.shields.io/codecov/c/github/open-telemetry/opentelemetry-java-instrumentation?style=for-the-badge">
+  </a>
+  <a href="releases">
```
Comment (mateuszrzeszutek) [inline README.md:23]: Wouldn't https://github.com/open-telemetry/opentelemetry-java-instrumentation/releases be safer? Absolute URL will work both from the repo view and the file view

File: README.md (hunk 1 above)
Comment (flands) [reply README.md:16]: Yup fixing

**Review by mateuszrzeszutek — APPROVED** [review]

**Pull request merged**